CHANGELOG-FORK.md

```diff
@@ -0,0 +1,69 @@
+# Changelog
+Changes in comparison to the upstream driver, forked at version REL42.2.5.
+
+## [Unreleased]
+
+## [x.x.x] (2023-05-04)
+- Release [CrateDB JDBC driver] 2.7.0, including the recent updates
```

Comment:
Why do we want to track here Crate JDBC release notes?
Track our changes at the fork is a good idea indeed! But I would keep the scope really on the fork and not related to were the fork is used and when/how Crate JDBC was released.

Reply:
Thanks. I've removed those items.